as-controller-board-status/routes/controller.go

```diff
@@ -40,16 +40,14 @@ func (c *Controller) AddAllRoutes() error {
 // GetStatus is a REST API endpoint that enables a web UI or some other downstream
 // service to inquire about the status of the upstream Automated Checkout hardware interface(s).
 func (c *Controller) GetStatus(writer http.ResponseWriter, req *http.Request) {
-	utilities.ProcessCORS(writer, req, func(writer http.ResponseWriter, req *http.Request) {
```

Comment:
I think we still want to keep CORS feature but using EdgeX's one  see some code refs here: https://github.com/edgexfoundry/go-mod-bootstrap/blob/8f4f829fcf456d393e71ac0eebf3660ad18169ea/bootstrap/handlers/httpcors.go#L29

Reply:
Yes, that can be enable if needed, but we don't have a UI that needs it, so leave that up to the user to enable it. Should add how in the documentation.

Reply:
@lenny-intel do you have example of how to enable it so I can add it to doc

Reply:
okay

Reply:
never mind, Jim sent me a "how to" from edgex-docs, I will follow that, this PR is ready to merge, thanks